workos/_base_client.py

```diff
@@ -19,31 +19,41 @@ class BaseClient(Protocol):
     _http_client: BaseHTTPClient
 
     @property
-    def audit_logs(self) -> AuditLogsModule: ...
+    def audit_logs(self) -> AuditLogsModule:
```

Comment:
... do we have different configs for `black`? Or different versions?

Reply:
I updated the version of black in the repo. I believe if you try reinstalling the version matching the setup.py you, me and CI will be in sync.

Reply:
Yup, needed to `pip install -U black`